### PR TITLE
Asyncio

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: mpd/asyncio.py [hunk 1]
```diff
@@ -273,82 +272,33 @@ async def __read_output_line(self):
...
-#            yield obj
-
-    def _parse_objects_direct(self, lines, delimiters=None, lookup_delimiter=False):
-        # This is a workaround implementing the above comment on Python 3.5. It
-        # is recommended that the commented-out code be used for reasoning, and
-        # that changes are applied there and only copied over to this
-        # implementation.
-
-        outerself = self
-        class WrappedLoop:
```
Comment (Mic92) [inline mpd/asyncio.py:310]: @chrysn @xcant I just uncomment the old code now that 3.5 is dropped as supported version. @xcant please check if https://github.com/Mic92/python-mpd2/pull/117 is not re-introduced.

**Comment by ochurlaud** [comment]
I was going to send the exact same patch to fix #135 that I just created.

**Comment by chrysn** [comment]
Looks good, and all changes I found to the expanded code were also applied inside (except for the ones where I didn't expand everything correctly).

I looked at the code but did not test the changes; is this code well enough covered, or are you looking for feedback on a test run too?

File: mpd/asyncio.py (hunk 1 above)
Comment (xcant) [reply mpd/asyncio.py:310]: I just cloned the repository and it works fine with the code where I detected the error, so #117 is not re-introduced.

**Comment by chrysn** [comment]
README still contains a reference to 3.4, should be 3.6 now.

**Comment by chrysn** [comment]
In my parallel approach I also added 1fe9f2532df3e7e603302ed9c8487ce547193f0f

**Comment by Mic92** [comment]
Thanks! I will rely on your branch.

**Pull request closed without merging**